index.js

```diff
@@ -384,7 +384,7 @@ function signApplicationAsync (opts) {
         })
           .then(function () {
             debuglog('Signing... ' + opts.app)
-            execFileAsync('codesign', args.concat('--entitlements', opts.entitlements, opts.app))
```

Comment:
@develar This may have caused a branching in async methods with thenables I think. 😕